src/main/java/org/folio/innreach/util/JsonHelper.java

```diff
@@ -53,4 +58,17 @@ public <T> T fromJson(InputStream inputStream, Class<T> valueType) throws IOExce
     return obj;
   }
 
+
+  public static Long getCheckOutTimeDuration(List<ConfigurationDTO> configData) {
+    log.info("JsonHelper:getCheckOutTimeDuration started");
+    long checkOutTime = 1;
+    if(!configData.isEmpty()) {
```

Comment:
It looks like if `configData` is empty it will default to 1 minute. Is that acceptable to us? In other words what are the consequences of having this value be 1 minute? Would the virtual items be deleted prematurely? Would it be better to throw an exception? Should the default be longer?

Another way of asking the question is: is this a _required_ configuration value?

If the config value isn't required and we do have a default, that default value should be defined in a constant like `DEFAULT_CHECKOUT_TIME_DURATION_MILLISECONDS`.